rpcd/playbooks/roles/rpc_support/defaults/main.yml

```diff
@@ -66,6 +66,9 @@ holland_git_repo_plugins:
 
 holland_pip_wheel_name: holland
 
+holland_packages:
+  - git
```

Comment:
This could be improved in a later commit, but do we track this work, in another card, as a low hanging fruit item?